pyproject.toml

```diff
@@ -213,4 +214,10 @@ setenv =
 commands = coverage combine
   coverage report -m
   coverage xml
+
+[testenv:ui]
+use_develop = false
+extras = testing
```

Comment:
```suggestion
```

Already declared on line 198.  Should be inherited if I understand https://tox.wiki/en/latest/user_guide.html#test-environments correctly.